packages/npm-dependencies-validation/src/index.ts

```diff
@@ -0,0 +1,90 @@
+import { spawnCommand } from "./npmUtil";
+import { resolve, join } from "path";
+import { access, readFile } from "fs/promises";
+import { constants } from "fs";
+import { DependencyIssue, IssueType } from "./types";
+
+const LS_DEPS = ["ls", "--depth=0"]; // dependencies and extraneous modules
```

Comment:
should theses two constants be with an "ARGS" suffix instead of "DEV"?

Reply:
comment and explain the `--depth=0` argument (direct deps only)

Reply:
Done